README.md

```diff
@@ -35,79 +39,36 @@ or
 yarn add @openfeature/js-sdk
 ```
 
-## Usage
-
-To configure the SDK you'll need to add a provider to the `OpenFeature` global signleton. From there, you can generate a `client` which is usable by your code. While you'll likely want a provider for your specific backend, we've provided a `NoopProvider`, which simply returns the default value.
-
-```typescript
-import { OpenFeature } from '@openfeature/js-sdk';
-
-// configure a provider
-OpenFeature.setProvider(new YourProviderOfChoice());
-
-// create a client
-const client = OpenFeature.getClient('my-app');
-
-// get a bool value
-const boolValue = await client.getBooleanValue('boolFlag', false);
+See [README.md](./packages/server/README.md)
 
-// get a string value
-const stringValue = await client.getStringValue('stringFlag', 'default');
+## Client
```

Comment:
What do you think about removing this section until we've verified that everything was published propertly?

Reply:
I think we should make it clear that this section is experimental and not ready for production use.

Reply:
I'm fine with that.

Reply:
I've removed this for now.